util/src/lib.rs

```diff
@@ -167,3 +167,59 @@ pub use io::*;
 pub use log::*;
 pub use kvdb::*;
 
+#[cfg(test)]
+mod tests {
```

Comment:
Those tests were not compiling in `bigint` crate - they are for `H256` which is implemented in `util/src/hash.rs`